modules/admin_manual/pages/release_notes.adoc

```diff
@@ -31,6 +32,70 @@
+* ownCloud Server and ownCloud Web are configured as outlined in the https://owncloud.github.io/clients/web/deployments/oc10-app/[documentation].
+
+=== Other Notable Changes
+
+* Federated shares can now also have an expiration date (including default and enforcement options) https://github.com/owncloud/core/pull/37548[#37548]
+* SGI images can now be displayed and have thumbnails https://github.com/owncloud/core/pull/37758[#37758]
+* When sharing public links via mail, the subject will now be translated https://github.com/owncloud/core/issues/37321[#37321]
+* Stability improvements for file checksums with versioning https://github.com/owncloud/core/pull/37934[#37934]
+* Manual file locking on the web UI can now be enabled/disabled in the admin settings section "Additional" https://github.com/owncloud/core/pull/37720[#37720]
+* Fixes and library updates for the Google Drive external storage integration https://github.com/owncloud/core/issues/25826[#25826] https://github.com/owncloud/core/pull/37739[#37739] https://github.com/owncloud/core/pull/37912[#37912]
+* When disabling storage encryption, the configuration is now properly cleaned up https://github.com/owncloud/core/pull/35980[#35980]
+* Many issues around unavailable federated shares have been fixed (e.g., not being able to remove them) https://github.com/owncloud/core/pull/38042[#38042] https://github.com/owncloud/core/pull/37956[#37956]
+* Performance improvements for the "Shared by link" view https://github.com/owncloud/core/pull/38000[#38000] https://github.com/owncloud/core/pull/38053[#38053]
+* Files with names up to 255 characters can now be shared via federation (was limited to 64 before) https://github.com/owncloud/core/issues/36730[#36730]
+* A user's language preference will not be overriden by locales sent by browsers anymore https://github.com/owncloud/core/pull/38073[#38073]
+* Fixed display of public links and user/group shares in case avatars are disabled https://github.com/owncloud/core/pull/37945[#37945] https://github.com/owncloud/core/pull/37964[#37964]
+* Fixed some translations not working with themes https://github.com/owncloud/core/pull/38072[#38072]
+* Improved output on errors for LDAP user sync https://github.com/owncloud/core/pull/37951[#37951]
+* A new occ command `files:troubleshoot-transfer-ownership` has been added to help finding issues with `files:transfer-ownership` https://github.com/owncloud/core/pull/37950[#37950]
+* Added values to the invalid uid list to prevent creating users with reserved UIDs https://github.com/owncloud/core/pull/37766[#37766]
+* The log level of "file locked" exceptions has been reduced to "debug" https://github.com/owncloud/core/pull/37907[#37907]
+
+=== Bundle and Delivery
+
+Since ownCloud Server 10.5 https://doc.owncloud.com/server/admin_manual/release_notes.html#changes-in-delivery[all supported apps are shipped in the ownCloud Server Complete bundles]. Below you find changes made to the bundle for Server 10.6.
```

Comment:
```suggestion
Since ownCloud Server 10.5 https://doc.owncloud.com/server/admin_manual/release_notes.html#changes-in-delivery[all supported apps are shipped in the ownCloud Server Complete bundles]. The following changes have been made to the bundle for Server 10.6:
```